tests/unit_tests/utils/test_general.py

```diff
@@ -807,6 +807,22 @@ def test_get_list_of_files_from_command_line(tmp_test_directory) -> None:
         gen.get_list_of_files_from_command_line(file_names, suffix_list)
 
 
+def test_generate_list_of_files():
+    with pytest.raises(ValueError, match=r"^No file list provided"):
+        gen.generate_list_of_files(None)
+
+    assert gen.generate_list_of_files("LICENSE") == [Path("LICENSE")]
+    yml_list = gen.generate_list_of_files("tests/resources/*.yml")
+    assert len(yml_list) > 0
+    yml_and_ecvs_list = gen.generate_list_of_files(
+        ["tests/resources/*.yml", "tests/resources/*.ecsv"]
+    )
+    assert len(yml_and_ecvs_list) > len(yml_list)
```

Comment:
Change to `>=` in case we decide to abandon all ecsv files in the future? Might make this test less safe though.

Reply:
I prefer to leave it, as the purpose of the test is to check if a list of wildcard pattern works.